plugins/out_es/es_conf.c

```diff
@@ -73,6 +174,13 @@ struct flb_elasticsearch *flb_es_conf_create(struct flb_output_instance *ins,
         return NULL;
     }
 
+    /* handle cloud_auth */
+    tmp = flb_output_get_property("cloud_auth", ins);
+    if (tmp) {
+        cloud_auth = (char *) tmp;
```

Comment:
do you need this? I think you can call `set_cloud_credentials(ctx, (char *) tmp);` directly?